lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -12,6 +12,8 @@
 from matplotlib import cm, rcParams
 from matplotlib.backends.backend_pdf import PdfPages
 from matplotlib import pyplot as plt
+from matplotlib.testing.determinism import (_test_source_date_epoch,
```

Comment:
I do not think it is a good idea to split tests into multiple files (apparently this will be the first in the whole matplotlib)